Give slightly clearer information as to why a topic was declined.

ValwareIRC opened the pull request:
I tried to set a topic with bold in it and didn't understand why it was rejected for some time.
I think this would be handy.

ValwareIRC committed:
Give correct information when declining a topic because of control chars.

ValwareIRC committed:
Update function.c

ValwareIRC committed:
update a bit more ♥♥

ltc-data commented on modules/chanserv/topic.c:
```diff
@@ -57,6 +57,11 @@ cs_cmd_topic(struct sourceinfo *si, int parc, char *parv[])
 	}
+	if (!validtopic_ctrl_chars(topic))
+	{
+		command_fail(si, fault_badparams, _("The new topic may not contain control chars, including colour, bold, italic and reverse."));
```
```suggestion
		command_fail(si, fault_badparams, _("The topic may not contain specific control chars, these include color, bold, italic and reverse."));
```

ValwareIRC committed:
Update modules/chanserv/topic.c

Please change color to colour to match the rest of the language as it uses British English.

Co-authored-by: Data <25669462+ltc-data@users.noreply.github.com>
